### DISCUSSION TITLE
Local file path question

### DISCUSSION THREAD

**Discussion by G0nz0uk** [grafana/grafana-csv-datasource#36] [Q&A] [answered]
Hello,

I think I'm being silly today, what am I doing wrong here?

![image](https://user-images.githubusercontent.com/15994497/108198500-9733be00-7113-11eb-838f-b159fb4c01a2.png)

tried:
/usr/local/bin/csv/NaviSet.csv
./usr/local/bin/csv/NaviSet.csv
usr/local/bin/csv/NaviSet.csv

**Comment by marcusolsson** [comment]
I believe it should be:

```
/usr/local/bin/csv/NaviSet.csv
```

Assuming you've already tried that, are you certain you have the permissions to access the file?

**Reply by G0nz0uk** [reply]
Permissions show as:

drwxr-xr-x 2 root root     4096 Feb 17 11:23 csv

Grafana is running in docker so maybe the wrong account?

**Reply by G0nz0uk** [reply]
Tried putting the csv in the home folder too, I wonder it docker can't get to it rather than permissions?

**Comment by G0nz0uk** [comment, marked as answer]
Also tried this to use the local csv in my test lab docker compose setup with no luck, am I missing something obvious?

As I'm using volume mounts I thought I'd dump a test123.csv here

![image](https://user-images.githubusercontent.com/15994497/108252514-a5ec9600-7150-11eb-835a-b2fd5f36e79d.png)

put it here

![image](https://user-images.githubusercontent.com/15994497/108252860-054aa600-7151-11eb-8bd7-06a7ffa9d597.png)

Test

![image](https://user-images.githubusercontent.com/15994497/108252924-17c4df80-7151-11eb-97b1-518119a2eb8a.png)

**Reply by marcusolsson** [reply]
Another user has reported the same thing. I've created an issue where we can try to track it down: https://github.com/marcusolsson/grafana-csv-datasource/issues/37

**Reply by G0nz0uk** [reply]
Solved it, in my docker compose yml file I had to bind a local volume to the Grafana Docker container.

So on the host I put the csv in /usr/share/csv folder added the below to the yml 

- /usr/share/csv:/etc/grafana/csv

Now when I bash into the Grafana container it sees the csv as local.  I then add the datasource as /etc/grafana/csv/test123.csv and it shows up.

**Reply by marcusolsson** [reply]
Glad it worked out! 🎉

**Comment by LuciferGreyInc** [comment]
Not sure if this help but I 'cd' into folder I'm using and then 'pwd' to get the folder path and then attached the filename to the end. From my server:

ubuntu:/sysStatLogs$ pwd
/home/ubuntu/sysStatLogs
ubuntu:/sysStatLogs$ ls
statsCPU.csv  statsRAM.csv
ubuntu:/sysStatLogs$

Which ends up with a local file string of '/home/ubuntu/sysStatLogs/statsCPU.csv' and that works for me.  The key was to make sure to grab all of the folders in the file path
